modules/cold-extraction/ColdDataRetriever.py

```diff
@@ -511,45 +523,68 @@ def run_cold_extraction():
 if __name__ == "__main__":
     config = defaultdict(lambda: None)
     # Read Default config.json file
-    with open('config.json', 'r') as f:
-        tmp_config = json.load(f)
-        config.update(tmp_config)
-
-    # CLI Argument Parser
     ap = argparse.ArgumentParser()
-
-    ap.add_argument("--NifflerSystem", default=config['NifflerSystem'],
+    try:
+        with open('config.json', 'r') as f:
+            tmp_config = json.load(f)
+            config.update(tmp_config)
+            
+        ap.add_argument("--NifflerSystem", default=config['NifflerSystem'],
                     help="Path to json file with Niffler System Information.")
-    ap.add_argument("--StorageFolder",
```

Comment:
I tried removing this. But I could not do it. I have put this block of code in try block.  Change of indentation might be causing this.